docs/architecture/adr-010-optimistic-message-size-independent-inclusion-proofs-and-pfd-fraud-proofs.md

```diff
@@ -0,0 +1,94 @@
+# ADR 010: Optimistic Message Size Independent Inclusion Proofs and PFD Fraud Proofs
```

Comment:
[optional]

```suggestion
# ADR 011: Optimistic Blob Size Independent Inclusion Proofs and PFB Fraud Proofs
```

- `010` => `011` to avoid a conflict with https://github.com/celestiaorg/celestia-app/pull/1005 . If you'd rather keep the 010 number, I can update https://github.com/celestiaorg/celestia-app/pull/1005 to use 011.


Accepting this suggestion implies a document wide rename of 
- `PFD` => `PFB` because https://github.com/celestiaorg/celestia-app/pull/978 merged
- `message` => `blob`